python/orca/src/bigdl/orca/learn/tf2/tf_runner.py

```diff
@@ -370,6 +376,12 @@ def step(self, data_creator, epochs=1, batch_size=32, verbose=1,
         if callbacks:
             replaced_log_dir = process_tensorboard_in_callbacks(callbacks, "fit", self.rank)
 
+        if self.model is None:
+            invalidInputError(False,
+                              "The model has not yet been created. "
+                              "Please input a model_creator when creating estimator "
+                              "or use load function of the estimator to load a model")
```

Comment:
Can we put this patch into a utility/static function to avoid duplicate code in these functions?

Reply:
I think we can only duplicate it...